manager/testdata/new-blueprints/scenario2/plotter.yaml

```diff
@@ -0,0 +1,206 @@
+                  object: data.parq
+              dataformat: parquet
+            api:
+              service:
+                assetID: m4d-notebook-sample/paysim
+                interfaceDetails:
+                  protocol: arrow-flight
+                  dataformat: arrow
+            actions:
+            - action: redact
+              column: field1
+  - name: flow-users
+    flowType: read
+    dataSetID: "m4d-notebook-sample/users"
+    subflows:
+    - name: subflow-users
+      trigger: timer
+      steps:
+      - - name: step1
+          cluster: thegreendragon
+          template: read-and-transform
+          parameters:
+            source:
+              dataSetId: "m4d-notebook-sample/users"
+              credentials:
+                read:
+                  vault:
+                    address: http://vault.m4d-system:8200
+                    authPath: /v1/auth/kubernetes/login
+                    role: module
+                    secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+              connection:
+                s3:
+                  endpoint: localhost:8001
+                  bucket: srcbucket
+                  object: users.parq
+              dataformat: parquet
+            api:
+              service:
+                assetID: m4d-notebook-sample/users
+                interfaceDetails:
+                  protocol: arrow-flight
+                  dataformat: arrow
+            actions:
+            - action: redact
+              column: blood_group
+  - name: flow-inventory
+    flowType: read
+    dataSetID: "m4d-notebook-sample/inventory"
+    subflows:
+    - name: subflow-inventory
+      trigger: timer
+      steps:
+      - - name: step1
+          cluster: thegreendragon
+          template: read-and-transform
+          parameters:
+            source:
+              dataSetId: "m4d-notebook-sample/inventory"
+              credentials:
+                read:
+                  vault:
+                    address: http://vault.m4d-system:8200
+                    authPath: /v1/auth/kubernetes/login
+                    role: module
+                    secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+              connection:
+                s3:
+                  endpoint: localhost:8001
+                  bucket: srcbucket
+                  object: inventory.parq
+              dataformat: parquet
+            api:
+              service:
+                assetID: m4d-notebook-sample/inventory
+                interfaceDetails:
+                  protocol: arrow-flight
+                  dataformat: arrow
+            actions:
+            - action: redact
+              column: field2
+  templates:
+  - name: read-and-transform
+    modules:
+    - name: arrow-flight-read
+      kind: M4DModule
+      confChart:
+        name: ghcr.io/mesh-for-data/read-module:0.1.0
+    - name: transform-conf
+      kind: M4DModule
```

Comment:
@roee88 The way I understand the transformation config module is that it injects a binary function. (e.g. wasm) into the arrow-flight-service. So it would not have an api that it exposes. Is that correct?

Reply:
@froesef Correct, so no api needed